### PR TITLE
K8 OVA Release Instructions for PI-5

### PR DESCRIPTION
**Pull request by mukul-seagate11**
### Describe your changes in brief
K8 OVA Release Instructions for PI-5

### Changes
 - [ ] Why is this change required? What problem does it solve?
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [ ] Please describe in detail how you tested your changes.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [x] tested locally
 - [ ] added new dependencies
 - [x] updated the docs
 - [ ] added a test

-----
[View rendered doc/ova/2.0.0/PI-5/CHANGELOG.md](https://github.com/Seagate/cortx/blob/k8-ova-pi5/doc/ova/2.0.0/PI-5/CHANGELOG.md)
[View rendered doc/ova/2.0.0/PI-5/S3_IO_Operations.md](https://github.com/Seagate/cortx/blob/k8-ova-pi5/doc/ova/2.0.0/PI-5/S3_IO_Operations.md)

### CODE REVIEW

File: doc/ova/2.0.0/PI-5/S3_IO_Operations.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+## Procedure
+
+- Run the following commands:
+```bash
+mkdir -p /var/log/cortx/auth
+touch /var/log/cortx/auth/s3iamcli.log
+kubectl get pods -o wide | grep cortx-data-pod
+```
+
+- Edit */etc/hosts* file and copy the IP of service data pod i.e. `cortx-data-pod` on same OVA image
```
[the remark this answers is not in the document]
Comment (mukul-seagate11) [reply doc/ova/2.0.0/PI-5/S3_IO_Operations.md:20]: ignore as its a file

File: doc/ova/2.0.0/PI-5/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+## Bug Fixes
+
+- init and data containers are not ordered across the pods[EOS-26888]
+- Support hctl status over http command[EOS-26482]
+- Modify consul endpoint handling to connect to endpoint with tcp protocol[EOS-26729]
+- hctl status takes too much time in containers[EOS-25703]
+- Reduce mkfs time for multiple io services[EOS-25688]
+- Handle Motr POD failures[EOS-23426]
```
[the remark this answers is not in the document]
Comment (mukul-seagate11) [reply doc/ova/2.0.0/PI-5/CHANGELOG.md:11]: ignore as its a functionality failure

**Review by pranav-sahasrabudhe — APPROVED** [review]
Approved

File: doc/ova/2.0.0/PI-5/CORTX_on_Open_Virtual_Appliance_PI-5.rst [hunk 3]
```diff
@@ -0,0 +1,86 @@
+
```
Comment (johnbent) [inline doc/ova/2.0.0/PI-5/CORTX_on_Open_Virtual_Appliance_PI-5.rst:1]: If this is the new OVA instructions, then this PR must also include an update to 
https://github.com/Seagate/cortx/blob/main/QUICK_START.md
to update the OVA link in that page.

**Review by johnbent — CHANGES_REQUESTED** [review]

File: doc/ova/2.0.0/PI-5/CORTX_on_Open_Virtual_Appliance_PI-5.rst (hunk 3 above)
Comment (mukul-seagate11) [reply doc/ova/2.0.0/PI-5/CORTX_on_Open_Virtual_Appliance_PI-5.rst:1]: done, it has been fixed

**Review by pranav-sahasrabudhe — APPROVED** [review]

**Review by shailesh-vaidya — APPROVED** [review]

File: QUICK_START.md [hunk 4]
```diff
@@ -3,7 +3,7 @@
 This document provides quick access to several different ways to set up CORTX for installation, testing, running, or deployment. CORTX is available in multiple formats, including as a pre-built virtual machine image - the easiest and fastest way to get CORTX up and running. Alternatively, CORTX can be built into a binary format from source code.  Finally, for those interested in running different configurations of CORTX, there are a variety of different options available. These include everything from the full installation to more slimmed down installations, such as configuring and running motr, the core object storage layer, on its own.  We have attempted to create easy-to-follow instructions for all of these installation/test methods which are organized below in order of difficulty.
  
  1. Download a CORTX virtual machine image to get up and running in minutes
-    1. On a single node: [LINK](doc/ova/2.0.0/PI-4/CORTX_on_Open_Virtual_Appliance_PI-4.rst)
+    1. On a single node using Kubernetes: [LINK](doc/ova/2.0.0/PI-5/CORTX_on_Open_Virtual_Appliance_PI-5.rst)
```
Comment (johnbent) [inline QUICK_START.md:6]: Do these instructions also allow you to run on multiple nodes?

**Pull request merged**